site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -165,18 +165,41 @@ We need a firewall to allow UDP traffic to nodes tagged as `game-server` via por
 next section.
 
 ```bash
-gcloud compute firewall-rules create game-server-firewall \
+gcloud compute firewall-rules create gke-agones-game-server-firewall \
   --allow udp:7000-8000 \
   --target-tags game-server \
   --description "Firewall to allow game server udp traffic"
 ```
 
 ## Creating the cluster
 
-Create a GKE cluster in which you'll install Agones. You can use
+Create a GKE cluster (preferably an Autopilot mode cluster) in which you'll install Agones. You can use
```

Comment:
Why is it preferable to use Autopilot?

There are tradeoffs, but why is it preferable?

This feels like we need a section on why choose one over the other? Or link to external docs?

Reply:
Okok sg!

Reply:
I think this is still using the same language?

Reply:
Forgot to remove that, thanks!